src/main/java/com/vasivkov/bootstrap/H2ServletInitializer.java

```diff
@@ -13,16 +17,21 @@
     private static final String[] H2_WEB_SERVER_ARGS = {"-web", "-webAllowOthers", "-webPort", "8082"};
     private Server tcpServer;
     private Server webServer;
+    private static final Logger LOGGER = Logger.getLogger(H2ServletInitializer.class.getName());
 
     @Override
-    public void contextInitialized(ServletContextEvent e) {
+    public void contextInitialized(ServletContextEvent servletContextEvent) {
```

Comment:
This method and class is responsible now for two things: logger initialization and h2 startup, so class name is now confusing. I think it's better either to have two separate ServletInitializers classes or to rename the class